ocviapy/__init__.py

```diff
@@ -664,14 +664,16 @@ def check_ready(self):
         return False
 
     def _check_with_periodic_log(self):
+        current_time = time.time()
+
         if self.check_ready():
             return True
 
-        if time.time() > self._time_last_logged + 60:
+        if current_time > self._time_last_logged + 60:
             self._time_remaining -= 60
```

Comment:
another `int` explicit conversion

also what's the `60` hardcoded value about , I see it added and then subtracted from _time_remaining ? if it's some sort of "security margin" ? if so, I'd strongly recommend to explicitly set so in the form of a variable with name that indicates so for better clarity 
```suggestion
        if current_time > int(self._time_last_logged + 60):
            self._time_remaining -= 60
```